### PR TITLE
changes for first point release (0.1)

### PR DESCRIPTION
**Pull request by aliceriot**
I haven't released it to npm yet, but the current beta release (0.0.7) is identical. When we merge this I'll push 0.1

### CODE REVIEW

**Comment by aliceriot** [comment]
@noisecapella would you mind taking a quick look at this? just a few changes that I made as I was working on [integrating this into MM](https://github.com/mitodl/micromasters/pull/3356)

File: package.json [hunk 1]
```diff
@@ -25,9 +28,10 @@
   "license": "ISC",
   "dependencies": {
     "babel-polyfill": "^6.23.0",
+    "babel-runtime": "^6.23.0",
```
Comment (noisecapella) [inline package.json]: What is this for?

**Comment by noisecapella** [comment]
Just the one question, looks pretty straightforward

File: package.json (hunk 1 above)
Comment (aliceriot) [inline package.json]: the docs are [here](https://babeljs.io/docs/plugins/transform-runtime/). basically what it does is ensure that there is less redundancy in the 'helper' functions that babel uses when transpiling code (i.e. some things can't be transformed to plain ES5 code without some sort of helper function, such as generators and the like) as well as polyfills for things. It also ensures that all polyfills and such do not pollute the global namespace, so if someone wants to use this library and they are using a different polyfill for, say, `Map`, than the one included with `babel-polyfill`, our code will not pollute the global namespace and should be able to exist alongside their existing, polyfilled code.

Comment (aliceriot) [inline package.json]: the babel-runtime plugin I think I just needed as a peer dependency

**Pull request merged**